### ISSUE TITLE
Mappings of municipalities to other region types for history data

### ISSUE THREAD

**Issue by jukop** [rOpenGov/geofi#28]
It would be useful in many cases to have mappings of municipality -> region and municipality -> subregion(seutukunta) from years much earlier than now. E.g. register-based data from HILMO care register contains municipality data from early 1970s, while currently geofi contains mappings for the 2013-2020.

I wonder if some resource/database already holds the borderline data of municipalities for 1970/1972-2012. Personally I had to use data from here in my past study: [Kuntaliitokset wikipedia](https://fi.wikipedia.org/wiki/Luettelo_kuntaliitoksista_Suomessa_kunnittain).

For weighting the study data with population data the borderline data are not necessarily needed so it would be useful to have more rough data (=no borderlines) available as well. That kind of mapping tables could be build based on the link I provided.

**Comment by antagomir** [comment]
Yes. If these are not readily available from established official sources, it would be justified to construct them manually and share as part of the package. Also good to check if we could/should utilize [avoindata.fi](https://www.avoindata.fi/fi) portal.

@teelmo also has some stuff nicely collected in [Github](https://github.com/teelmo/geodata) but not really historical ones.

**Comment by pitkant** [comment]
I think this would be useful as well. I added functionalities and datasets related to this into the sorvi package 0.8.20.9000 experimental version:

https://github.com/rOpenGov/sorvi/

It is based on work done in FinnONTO on the Finnish Spatio-temporal Ontology, I have translated the RDF/Turtle files into a clean dataset that is easy to use in R. I think this, along with mappings to other types of administrative divisions such as lääni, kihlakunta, tuomioistuinpiiri etc., would be very useful if it could be adopted in some form into the geofi package!

**Comment by antagomir** [comment]
Interesting! To be tested & discussed in more detail.

**Comment by pitkant** [comment]
Should this be considered to be more part of a hypothetical "geofi-data" package / repository (as referred in #13 ) instead of geofi core package?

**Comment by antagomir** [comment]
Sounds feasible. However, it could make sense to develop and plan publications jointly
